openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -172,4 +172,17 @@ jQuery(function () {
     $('.column').disableSelection();
     $('.trash').disableSelection();
     $('#topNotice').hide();
+
+    // Functionality for check.html
+
+    $('.list-books a').click(function() {
```

Comment:
could you put this in a  file called add-book.js file which exports an init function

``
const addbook = require('./addbook.js');
addbook.init();
```

Reply:
Yes, no problem. Sorry that it took me a bit to get back to you. Thanks, @jdlrobson